fuzz/trinity.py

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env python
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+#
+# Copyright: 2016 IBM
+# Author: Praveen K Pandey <praveen@linux.vnet.ibm.com>
+#
+
+import os
+import re
+
+from avocado import Test
+from avocado import main
+from avocado.utils import archive, build, process
+from avocado.utils.software_manager import SoftwareManager
+
+
+class Trinity(Test):
+
+    """
+    This testsuite test  syscall by calling syscall
+    with random system call and varying number of args
+    """
+
+    def setUp(self):
+        '''
+        Build Trinity
+        Source:
+        https://github.com/kernelslacker/trinity
+        '''
+
+        process.run('useradd  trinity', sudo=True)
+
+        smm = SoftwareManager()
+
+        for package in ("gcc", "make"):
+            if not smm.check_installed(package) and not smm.install(package):
+                self.error(
+                    "Fail to install %s required for this test." % package)
+
+        locations = ["https://github.com/kernelslacker/trinity/archive/"
+                     "master.zip"]
+        tarball = self.fetch_asset("trinity.zip", locations=locations)
```

Comment:
@apahim  thanks for looking into this , will create a new pull request with this change 

Thanks